remote-processor/AnswerMessage.cpp

```diff
@@ -57,7 +59,8 @@ const string& CAnswerMessage::getAnswer() const
 // Status
 bool CAnswerMessage::success() const
 {
-    return getMsgId() == ESuccessAnswer;
+    /* FIXME this test is buggy as there are acutally 3 values in the enum... */
```

Comment:
I don't why it would be buggy... Currently, ESuccessAnswer is the only value that represents a success.